src/javascript/app/common/common_functions.js

```diff
@@ -8,6 +9,7 @@ const hideEU = () => {
             $('.eu-show').setVisibility(1);
             $('.eu-hide-parent').parent().setVisibility(0);
         }
+        $('.id-show').setVisibility(isIndonesia());
```

Comment:
if we only need this in platforms page, we don't need to call it in such a common function and can only call it in platforms page

Reply:
okay sure